packages/shorebird_cli/bin/shorebird.dart

```diff
@@ -61,6 +62,7 @@ Future<void> main(List<String> args) async {
         shorebirdValidatorRef,
         shorebirdVersionRef,
         xcodeBuildRef,
+        shorebirdAndroidArtifactsRef,
```

Comment:
Can we keep this list sorted alphabetically?

Reply:
I wish there were a linter that could check those things :*(